website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,49 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Selection
+
+A multi-select table includes checkboxes enabling users to select multiple rows in a table for purposes of performing bulk operations. Checking or unchecking the checkbox in the table header either selects or deselects the checkboxes on each row in the table body. Individual checkboxes in the rows can also be selected or deselected.
+
+Multi-selection consist of: 
+
+1. `Header Column / Select`: Used in the table's header row. This acts as the parent checkbox, allowing the selection or deselection of all child rows on a single page simultaneously.
+
+![Example of multi-selection in a table header](/assets/components/table/multi-select-header.png)
+
+2. `Cell / Select`: Used in each table row. It allows for individual row selection.
+
+![Example of multi-selection within table cells](/assets/components/table/multi-select-cells.png)
+
+!!! Info 
+
+In the Ember component, unlike the `cell / select` Figma component, achieving an indeterminate state is not possible. 
+!!!
+
+### Intended interaction
+
+- When individual rows are selected, the parent checkbox in the `Table` header changes to display an indeterminate state. 
+- When no or some rows (but not all) are selected on a single page, clicking the parent checkbox in the `Table` header will change it to display as checked, and all rows on that page will be selected.
+- When all rows are selected on a single page, the parent checkbox in the `Table` header appears as checked. Clicking the parent checkbox will deselect all rows on that page.
```

Comment:
```suggestion
- When all rows are selected on a single Table, the parent checkbox in the header appears as checked. Clicking the parent checkbox will deselect all rows on that page.
```

Ah, I see now why you're using the term "page", as in a paginated table. I think this might get a little confusing because this could refer to the page within the UI more holistically, or a page of results (out of multiple) within a table. I might recommend sticking to "Table" for now just for consistency and to not introduce confusion.